files/image_config/logrotate.d/rsyslog

```diff
@@ -30,14 +28,14 @@
 {
     rotate 4
     weekly
-    size 100M
+    maxsize 100M
     missingok
     notifempty
     compress
     delaycompress
     sharedscripts
     postrotate
-        invoke-rc.d rsyslog rotate > /dev/null
```

Comment:
It was not working properly. Apparently it's a bug: https://askubuntu.com/a/509557.

There are apparently two workarounds:
1. `service rsyslog rotate >/dev/null 2>&1 || true`
2. `kill -HUP $(cat /var/run/rsyslogd.pid)`

I have tested the latter solution. I did not try the former, as the `|| true` seems a bit hacky. I can test it if you'd like.